presto-docs/src/main/sphinx/functions/regexp.rst

```diff
@@ -115,3 +115,28 @@ with a few notable exceptions:
     array. Trailing empty strings are preserved::
 
         SELECT regexp_split('1a 2b 14m', '\s*[a-z]+\s*'); -- [1, 2, 14, ]
+
+.. function:: regexp_count(string, pattern) -> integer
+
+    Returns the number of times ``pattern`` occurs in ``string``::
+
+        SELECT regexp_count('1a 2b 14m', '\s*[a-z]+\s*'); -- 3
+
+.. function:: regexp_instr(string, pattern) -> integer
```

Comment:
The name `instr` seems unintuitive. How about `regexp_position`, as that matches the string `position` function?

Reply:
Thanks for your advice, I have changed the function name. *regexp_position* is more readable, I named it  'regexp_instr' before just because oracle uses this name.